src/test/addSourceWizard/addSourceWizard/hardCodedComponents/cost_management_azure.test.js

```diff
@@ -27,7 +27,9 @@ describe('Cost Management Azure steps components', () => {
     render(<Cm.SubscriptionID />);
 
     expect(
-      screen.getByText('Run the following command in Cloud Shell to obtain your Subscription ID and enter it below:')
+      screen.getByText(
+        'Run the following command in Cloud Shell to obtain the Subscription ID where the cost export is being stored and enter it below:'
```

Comment:
could it be split into two different steps:
* Run the following command in a Cloud Shell to get the Subscription ID and copy the result.
* Paste the result number in the *Subscription ID* field below.